pkg/apis/openebs.io/upgrade/v1alpha1/taskconfig.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
can we rename the file to config.go
we understand that this is related to upgrade.

Reply:
done.